java/java-ranger-regression/alarm/impl/B_ALARM_Functional_c_T.java

```diff
@@ -0,0 +1,54 @@
+
+/* Block signals for model 'ALARM_Functional' */
+
+public class B_ALARM_Functional_c_T {
```

Comment:
As discussed in #1009, this PR will need some Refactoring regarding the naming conventions.

Reply:
@mmuesly , thanks, I have refactored the benchmark to follow the Java naming conventions.